styles.css

```diff
@@ -0,0 +1,125 @@
+
+body {
+  background: #e5e5e5;
+  margin: 0;
+}
+
+.topbar {
+  display: flex;
+  background-color: #3c3a39;
+  color: #fff;
+  position: absolute;
+  width: 375px;
+  height: 92px;
```

Comment:
- Setting the maximum width of top bar will fixed the width to 375px. This will prevent your website to be responsive.

![port-1](https://user-images.githubusercontent.com/49184579/176735328-a5527a3e-091c-4996-8dc4-0987440bd23c.png)


See the picture above to compare your website with the figma design.
1. Font is not similar. It should be Crete Round font. Check the figma design.
2. White space in the website. This is because you have set your max-width of toolbar and headline sections to 375px and your background image is taking up space larger than 375px.
3. Please follow the figma design for all margin/padding, color, and fonts

Reply:
@aboongm Thank you for the review. Are you available to assist me in this